Python/TelemetrySystem/test_telemetry.py

```diff
@@ -1,11 +1,30 @@
 import unittest
 
-from telemetry import *
+from telemetry import TelemetryDiagnostics
+from mock_client import MockClient
+
 
 class TelemetryDiagnosticControlsTest(unittest.TestCase):
```

Comment:
I'd rather separate these tests, since they're not really connected with each other.

Reply:
I test the same class TelemetryDiagnostics. Why do I need to separate them ?

Reply:
The problem of the class is that there are several reasons to change it. For instance, if I want to modify check_transmission_diagnostic_message test, I need to modify it. If I want to modify check_transmission_other_message test I also have to modify it. Overall, Diagnostics class should run tests like test runner, but tests themselves are separate entities with different logic as far as I see.

Reply:
Sorryyy, thought that it's a separate class with logic, it's just unit tests =)